docs/xtrabackup-versioning-scheme.md

```diff
@@ -0,0 +1,11 @@
+### Percona XtraDB Cluster 5.7 versioning scheme:
+
+Here is what the numbers in the product version specification mean:
+
+For example, in Percona XtraDB Cluster 5.7.39-31.61
+
+* `5.7.39` - the version of Percona Server for MySQL that Percona XtraDB Cluster is based on
+
+* `31` - the WSREP API version that originates from WSREP_VERSION file
```

Comment:
I would leave only 
`31 - WSREP API version`
That's it. We can document internally which file keeps it and so on. It may change over the time, and the above info is enough for the user to understand what 31 is.

Reply:
Updated